metricflow/sql/sql_exprs.py

```diff
@@ -1144,3 +1149,64 @@ def matches(self, other: SqlExpressionNode) -> bool:  # noqa: D
         if not isinstance(other, SqlRatioComputationExpression):
             return False
         return self._parents_match(other)
+
+
+class SqlBetweenExpression(SqlExpressionNode):
+    """A BETWEEN clause like `column BETWEEN val1 AND val2`"""
+
+    def __init__(  # noqa: D
+        self, column_arg: SqlExpressionNode, left_value: SqlExpressionNode, right_value: SqlExpressionNode
```

Comment:
Shall we call these `min_value` and `max_value` instead of `left_value` and `right_value`?

I guess at this level we don't necessarily know if someone has done something silly and entered a config expr which will return FALSE regardless of input (e.g., `num_entries BETWEEN 10 AND 2`), so maybe `left` and `right` are less confusing overall.

Reply:
I did some searching, theres a bunch of variety. Examples,

```
- start_expr | end_expr
- start_value | end_value
- begin_expr | end_expr
```

I'm not the best at naming, but either of those feels fine to me (leaning towards `start_expr/end_expr`). Lemme know your thoughts

Reply:
start_expr/end_expr seems pretty reasonable to me!